src/Form/Form.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import classes from "./Form.module.css";
+import firebase from "firebase/app";
+import "firebase/firestore";
+
+const Form = () => {
+  const [name, setName] = useState("");
+  const [frequency, setFrequency] = useState(0);
+  const [date, setDate] = useState("");
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    let db = firebase.firestore();
+    let data = {
+      name,
+      frequency,
+      date
+    };
+    db.collection("userThree").add(data);
+    // userThree will be replaced by user token
+    // a new doc is created with each submission.
+  };
+
+  return (
+    <form onSubmit={e => handleSubmit(e)}>
+      <h3 className={classes.formTitle}>Add New Item</h3>
+      <input
+        className={classes.inputName}
+        type="text"
+        name="name"
+        placeholder="Item Name"
+        value={name}
+        onChange={e => setName(e.target.value)}
+        required
+      />
+      <div>
+        <label>
+          <input
+            type="radio"
+            name="frequency"
+            value="7"
+            onChange={e => setFrequency(e.target.value)}
```

Comment:
This is completely a suggestion, but cool to think about!

If you find you're doing the same thing over and over again with the `setFrequency` function like this, you can create one function that does this:

```js
function onChangeInput(e) {
  setFrequency(e.target.value);
}
```

Then use this function instead:
```js
<input
    type="radio"
    name="frequency"
    value="14"
    onChange={handleSetFrequency}
/>
```

Reply:
I'm not having luck testing out this alternative. Would we need to name the function `handleSetFrequency` instead of `onChangeInput`?

`function handleSetFrequency(e) {
    setFrequency(e.target.value);
}`

As well as pass it the event in the input's onChange?

`onChange={e => handleSetFrequency(e)}`

I may still leave it as is, but want to see if I can get your suggestion to work.